content/ekscloudwatchcontainerinsights/installwordpress.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Install Wordpress"
+chapter: false
+weight: 2
+---
+
+<h3>Install Wordpress to EKS Cluster</h3>
+  
+<img src="/images/ekscwci/wordpresslogo.png">
+
+We’ll be using the following Wordpress Distribution to install Wordpress to our EKS cluster. To install this we'll be using helm for an easy deployment method. 
+
+https://github.com/helm/charts/tree/master/stable/wordpress 
+
+In your Cloud9 Workspace terminal you just need to run the following command to deploy wordpress. 
+
+```
+helm install stable/wordpress 
+```
+
+You will see that this chart does a number of items. Including creating a persistent volume clain in EKS, create a Pod named after the release of Wordpress being installed (at time of this lab creation “understood-zebra” ) Multiple secrets stores and a stateful set. 
+ 
+<img src="/images/ekscwci/helminstalloutput.png">
```

Comment:
We shouldn't need the embedded html... can you swap to markdown format, and also move images under /static/cwci/ (or some other relevant directory name)?